easybuild/easyconfigs/m/matplotlib/matplotlib-2.1.0-foss-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = 'Bundle'
+
+name = 'matplotlib'
+version = '2.1.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://matplotlib.org'
+description = """matplotlib is a python 2D plotting library which produces publication quality figures in a variety of
+ hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python
+ and ipython shell, web application servers, and six graphical user interface toolkits."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('Tkinter', '%(pyver)s', versionsuffix),
+    ('libpng', '1.6.32'),
+    ('freetype', '2.8'),
+]
+
+exts_list = [
+    ('Cycler', '0.10.0', {
+        'modulename': 'cycler',
+        'source_tmpl': 'cycler-%(version)s.tar.gz',
+        'source_urls': ['https://pypi.python.org/packages/source/C/Cycler'],
+        'checksums': [
+            'cd7b2d1018258d7247a71425e9f26463dfb444d411c39569972f4ce586b0c9d8',  # cycler-0.10.0.tar.gz
+        ],
+    }),
+    (name, version, {
+        'source_urls': ['https://pypi.python.org/packages/source/m/matplotlib'],
+        'checksums': [
+            '4b5f16c9cefde553ea79975305dcaa67c8e13d927b6e55aa14b4a8d867e25387',  # matplotlib-2.1.0.tar.gz
+        ],
+    }),
+]
+
+# specify that Bundle easyblock should run a full sanity check, rather than just trying to load the module
+full_sanity_check = True
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
+}
```

Comment:
This is the default sanity check done by `PythonBundle`, so this can be dropped too.